HelpAssistHer/help-assist-her
# Add icons to resource card in results

## Description
There are quite a few changes in this PR, in order to get icons added. The changes include:
- Add an npm package that converts svg files to React components
- Add a README file with instructions to run the svgtoreact command
- Add the phone, globe, and location svgs, which are all used on the resource card in the results
- Add the associated React components for each of the new icons
- Add the icons to the resource card, so they are actually displayed
- Additional formatting/styling to match the designs

## Test Plan
1. Go to http://localhost:4000/mini-app/
2. Type "nyc" in the location and hit enter
3. Make sure each of the cards looks like this:

<img width="381" alt="screen shot 2018-10-29 at 3 16 40 pm" src="https://user-images.githubusercontent.com/6378435/47683719-adf22780-db8d-11e8-91bf-0685897a1c87.png">
